apps/charterafrica/src/assets/icons/Chevron-up, Size=24, Color=CurrentColor.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="10" viewBox="0 0 18 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M17 9L9 1L1 9" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
if CurrentColor
```suggestion
<path d="M17 9L9 1L1 9" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```

...otherwise this file should be named `...Black.svg`.